src/fetch/payouts.py

```diff
@@ -455,54 +528,29 @@ def construct_payouts(
             ),
         ),
         # Dune: Fetch Solver Slippage & Reward Targets
-        slippage_df=pandas.DataFrame(dune.get_period_slippage()),
+        slippage_df=slippage_df,
```

Comment:
Is there a reason you have a separate variable `slippage_df` for computing slippage but not one for `reward_target_df` ? Not an issue, but looks a bit inconsistent

Reply:
Good point. I mostly added it as a separate variable to be able to more easily comment it out and substitute it for an empty dummy slippage data frame.

I think it is a bit ugly to have those function calls inside arguments of function calls. But to keep changes minimal and consisent, I will just change it back.